function-reference/query-functions/text.xml

```diff
@@ -47,7 +47,17 @@ query variables and similar state.
   </tr>
   <tr>
     <ti>
-      <c>use_enable flag str val</c> 
+    <c>usex flag [true output] [false output] [true suffix] [false suffix]</c>
+    </ti>
+    <ti>
+      If flag is set, echo [true output][true suffix] (defaults to "yes"),
+      otherwise echo [false output][false suffix] (defaults to "no"). If unspecified,
+      the suffixes default to empty strings.
```

Comment:
From this wording it is not entirely clear that the yes/no defaults are for "true output" and "false output".